src/config.ts

```diff
@@ -113,6 +113,53 @@ export const getIntegratorIdFromLabel = (label: string): string | undefined => {
     }
 };
 
+/**
+ * The JSON config for each Market Maker, providing information including URLs, type of order supported and authentication.
```

Comment:
```suggestion
 * The JSON config for each Market Maker, providing information including URIs, type of order supported and authentication.
```

we typically say `makerUri` and not `makerUrl` - there's no real reason why to prefer Uri over Url, but now that we have it everywhere in our codebase, want to keep all mentions of it the same